Sources/PackageDescription4/SupportedPlatforms.swift

```diff
@@ -37,6 +37,9 @@ public struct Platform: Encodable {
     /// The Windows platform
     @available(_PackageDescription, introduced: 5.2)
     public static let windows: Platform = Platform(name: "windows")
+
+    /// The Android platform
+    public static let android: Platform = Platform(name: "android")
```

Comment:
Please annotate this with `@available(_PackageDescription, introduced: 5.2)`

Reply:
Done.